### ISSUE TITLE
migration to 4.0 yunohost

### ISSUE THREAD

**Issue by boistordu** [YunoHost-Apps/lufi_ynh#45]
board olimex lime 2
impossible to launch it anymore

```
-- The job identifier is 460490.
Aug 02 01:43:45 carton[30836]: ListUtil.c: loadable library and perl binaries are mismatched (got handshake key 0x9400080, needed 0x9380080)
Aug 02 01:43:45 systemd[1]: lufi.service: Control process exited, code=exited, status=1/FAILURE
-- Subject: Unit process exited
-- Defined-By: systemd
-- Support: https://www.debian.org/support
```

**Comment by ldidry** [comment]
Note to maintainers of the package: this just need a remove of `local` directory then redo a `carton install` with the options previously used.

**Comment by boistordu** [comment]
`sudo carton install --deployment --without=sqlite  --without=mysql --without=htpasswd --without=test`

**Comment by HugoPoi** [comment]
I confirm the problem on x86 too

Workaround
```bash
sudo apt install carton
cd /var/www/lufi
sudo sudo -u lufi carton install --deployment --without=sqlite --without=mysql --without=htpasswd --without=test
sudo yunohost app upgrade lufi -u https://github.com/YunoHost-Apps/lufi_ynh/tree/master
```

**Comment by tomdereub** [comment]
Hi,
For me, Lufi was working after upgrading to yunohost 4 and buster, but after a reboot the service couldn't start anymore, with the same error as @boistordu.
I tried @HugoPoi 's workaround, but didn't work for me, here is the log : https://plaf.fr/zerobin/?945545867cadd9f3#DBLMamJ6ARjXJ94uin3cCu1ja3RRGHsRMS3mL8NGHcJd
And as Lufi wouldn't start, it has been removed in the upgrade process, and pre-upgrade backup restoration failed.
Then I did a fresh installation, it's working, but restoration is still impossible.
